crd/src/lib.rs

```diff
@@ -27,11 +29,35 @@ impl Crd for KafkaCluster {
     const CRD_DEFINITION: &'static str = include_str!("../kafkaclusters.crd.yaml");
 }
 
-#[allow(non_camel_case_types)]
 #[derive(Clone, Debug, Deserialize, JsonSchema, Serialize)]
-pub enum KafkaVersion {
-    #[serde(rename = "2.6.0")]
-    v2_6_0,
+pub struct KafkaVersion {
+    kafka_version: String,
+    scala_version: Option<String>,
+}
+
+impl KafkaVersion {
+    pub const DEFAULT_SCALA_VERSION: &'static str = "2.13";
+
+    pub fn kafka_version(&self) -> &str {
+        &self.kafka_version
+    }
+
+    pub fn scala_version(&self) -> &str {
+        &self
+            .scala_version
+            .as_deref()
+            .unwrap_or(Self::DEFAULT_SCALA_VERSION)
+    }
+
+    pub fn get_fully_qualified_version(&self) -> String {
```

Comment:
If you want you could also remove this `get_` prefix.

Reply:
done